compiler/rustc_codegen_llvm/src/attributes.rs

```diff
@@ -301,20 +289,22 @@ pub fn from_fn_attrs(cx: &CodegenCx<'ll, 'tcx>, llfn: &'ll Value, instance: ty::
     // The target doesn't care; the subtarget reads our attribute.
     apply_tune_cpu_attr(cx, llfn);
 
-    let features = llvm_target_features(cx.tcx.sess)
```

Comment:
@lqd in answer to your question, we would call the `llvm_target_features` function for every single function call in the past, but it was somewhat(?) cheaper in that it would be able to produce an iterator of `&str`s, rather than a `Vec<String>` as `llvm_global_features` is forced to now.

However as you see in this area, we'd go on to immediately convert each of those `&str`s to a `String` anyway. As thus I wouldn't expect much difference in performance.

Reply:
makes sense, thanks for the answer !